.github/workflows/build.yml

```
@@ -17,7 +17,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: gradle/wrapper-validation-action@v2.1.2
+      - uses: gradle/wrapper-validation-action@v3.3.0
```

See 
> As of v3 this action has been deprecated by gradle/actions/wrapper-validation.
Any workflow that uses gradle/wrapper-validation-action@v3 will transparently delegate to gradle/actions/wrapper-validation@v3.

```suggestion
      - uses: gradle/actions/wrapper-validation@v3.3.0
```